Dockerfile.extra

```diff
@@ -0,0 +1,58 @@
+# -- plugins -------------------------------------------------------------------
```

Comment:
IIUC this file isn't currently used anywhere; I'd like to see how you plan to call into this from CI, otherwise it's not really possible to review this PR.

Reply:
It's documented in the commit message that introduces the file, and you can also see it in the `docker` CI workflow.

Reply:
I think you don't need the extra Dockerfile at all if you just add a stage at the very end that says `FROM production AS vast`.